### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by danabens**
minor readme updates

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,22 @@
-=====================
-SageMaker Experiments
-=====================
+.. image:: https://github.com/aws/sagemaker-experiments/raw/master/branding/icon/sagemaker-banner.png
```
Comment (wangweixun) [inline README.rst]: Page not found.

File: README.rst [hunk 2]
```diff
@@ -10,10 +26,10 @@ SageMaker Experiments is an AWS service for tracking machine learning Experiment
 Concepts
 --------
 
-- Experiment: A collection of related Trials. Add Trials to an Experiment that you wish to compare together.
-- Trial: A description of a multi-step machine learning workflow. Each step in the workflow is described by a TrialComponent.
-- TrialComponent: A description of a single step in a machine learning workflow. 
-- Tracker: A Python context-manager for logging information about a single TrialComponent.
+- **Experiment**: A collection of related Trials. Add Trials to an Experiment that you wish to compare together.
+- **Trial**: A description of a multi-step machine learning workflow. Each step in the workflow is described by a TrialComponent.
```
Comment (wangweixun) [inline README.rst]: Nit: The working seems to imply that trial components in a trial has an ordering.

File: README.rst [hunk 3]
```diff
@@ -10,10 +26,10 @@ SageMaker Experiments is an AWS service for tracking machine learning Experiment
...
+- **TrialComponent**: A description of a single step in a machine learning workflow.
```
Comment (wangweixun) [inline README.rst]: Nit: Mention a few examples here on what could be a trial component?

File: README.rst (hunk 1 above)
Comment (danabens) [reply README.rst]: the banner is part of this pull request see https://github.com/aws/sagemaker-experiments/pull/20/commits/a67bcb644e94d9354fb21025612311aa58854f4e

File: README.rst (hunk 2 above)
Comment (danabens) [reply README.rst]: updated

File: README.rst (hunk 3 above)
Comment (danabens) [reply README.rst]: added examples and link to more info

**Pull request merged**